package.json

```diff
@@ -35,7 +35,8 @@
         ],
         "setupFilesAfterEnv": [
             "@testing-library/jest-dom/extend-expect"
-        ]
+        ],
+        "testEnvironment": "jsdom"
```

Comment:
Global variable window was undefined without this, but jest still causes problems because the web crypto API is undefined. I think that I need to do a new PR to make changes to jest or maybe to the encrypt implementation